lib/grp.gd

```diff
@@ -4509,13 +4509,15 @@ DeclareGlobalFunction("GroupEnumeratorByClosure");
 ##  <C>LowIndexSubgroupsFpGroup</C>. In other cases, it uses repeated
 ##  calculation of maximal subgroups.
 ##  <P/>
+##  The operation <C>LowLayerSubgroups</C> works similar but does not bound
```

Comment:
While looking into tweaking the GAPDoc, I realized that `LowLayerSubgroups` actually is a global function, not an operation. Moreover, it takes additional (optional) arguments, and already has an  documentation comment in `grplatt.gd` (which also is in the manual). 

Perhaps the plan was/is to turn `LowLayerSubgroups` into an operation? In any case, the change as it is in this PR right now does not seem correct... @hulpke ?